docs/src/main/sphinx/security.rst

```diff
@@ -27,12 +27,13 @@ Authentication
   :maxdepth: 1
 
   security/authentication-types
-  security/ldap
   security/password-file
+  security/ldap
   security/salesforce
   security/oauth2
   security/kerberos
   security/cli
+  security/jwt
```

Comment:
This should go above, since it's an authentication method

Reply:
No. 
1. security/cli resolves to the title "CLI Kerberos authentication" and logically belongs under "Kerberos authentication."
2. Moreover, an upcoming PR will move the entire contents of security/cli into installation/cli so that it disappears from this Security chapter.